airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```diff
@@ -1,79 +1,271 @@
+    "taskInstance": "הערת משימה"
   },
   "pools": {
-    "deferred": "בהמתנה עד קיום תנאי",
+    "deferred": "נדחה לתזמון עתידי",
     "open": "פתוח",
-    "pools_one": "מאגר",
-    "pools_other": "מאגרים",
-    "queued": "בהמתנה למשאבים",
-    "running": "בריצה",
-    "scheduled": "בתזמון"
+    "pools_one": "מאגר משאבים",
+    "pools_other": "מאגרי משאבים",
+    "queued": "בתור",
+    "running": "בביצוע",
+    "scheduled": "מתוזמן"
+  },
+  "runId": "מזהה הרצה",
+  "runTypes": {
+    "asset_triggered": "הופעל על-ידי נכס",
+    "backfill": "השלמה למפרע",
+    "manual": "הפעלה ידנית",
+    "scheduled": "מתוזמן"
   },
+  "seconds": "{{count}} שניות",
+  "security": {
+    "actions": "פעולות",
+    "permissions": "הרשאות",
+    "resources": "משאבים",
+    "roles": "תפקידים",
+    "users": "משתמשים"
+  },
+  "selectLanguage": "בחר שפה",
+  "sourceAssetEvent_one": "אירוע נכס מקור",
+  "sourceAssetEvent_other": "אירועי נכס מקור",
+  "startDate": "תאריך התחלה",
+  "state": "מצב",
   "states": {
-    "deferred": "בהמתנה עד קיום תנאי",
-    "failed": "כשלון",
-    "no_status": "ללא סטאטוס",
-    "queued": "בהמתנה למשאבי ריצה",
+    "deferred": "נדחה",
+    "failed": "נכשל",
+    "no_status": "ללא סטטוס",
+    "none": "ללא סטטוס",
+    "queued": "בתור",
     "removed": "הוסר",
-    "restarting": "בהפעלה מחדש",
-    "running": "בריצה",
-    "scheduled": "בהמתנה לתזמון",
+    "restarting": "מופעל מחדש",
+    "running": "בביצוע",
+    "scheduled": "מתוזמן",
     "skipped": "דולג",
-    "success": "הצלחה",
-    "up_for_reschedule": "בהמתנה לתזמון פנימי",
-    "up_for_retry": "בהמתנה לניסיון נוסף",
-    "upstream_failed": "תלויות נכשלו"
+    "success": "הצליח",
+    "up_for_reschedule": "ממתין לתזמון מחדש",
+    "up_for_retry": "ממתין לניסיון חוזר",
+    "upstream_failed": "משימות קודמות נכשלו"
+  },
+  "switchToDarkMode": "עבור למצב כהה",
+  "switchToLightMode": "עבור למצב בהיר",
+  "table": {
+    "completedAt": "הושלם בתאריך",
+    "createdAt": "נוצר בתאריך",
+    "filterByTag": "סנן לפי תגית",
+    "filterColumns": "סנן עמודות",
+    "filterReset_one": "אפס מסנן",
+    "filterReset_other": "אפס מסננים",
+    "from": "מ-",
+    "maxActiveRuns": "מספר הרצות פעילות מרבי",
+    "noTagsFound": "לא נמצאו תגיות",
+    "tagMode": {
+      "all": "הכל",
+      "any": "כלשהו"
+    },
+    "tagPlaceholder": "סנן לפי תגית",
+    "to": "עד"
+  },
+  "task": {
+    "documentation": "תיעוד משימה",
+    "lastInstance": "משימה אחרונה",
+    "operator": "אופרטור",
+    "triggerRule": "כלל הפעלה"
+  },
+  "task_one": "משימה",
+  "task_other": "משימות",
+  "taskId": "מזהה משימה",
+  "taskInstance": {
+    "dagVersion": "גרסת Dag",
+    "executor": "מבצע",
+    "executorConfig": "הגדרות מבצע",
+    "hostname": "שם מארח",
+    "maxTries": "מספר נסיונות מרבי",
+    "pid": "מזהה תהליך",
+    "pool": "מאגר משאבים",
+    "poolSlots": "מכסת משאבים",
+    "priorityWeight": "משקל עדיפות",
+    "queue": "תור",
+    "queuedWhen": "זמן כניסה לתור",
+    "scheduledWhen": "זמן תזמון",
```

Comment:
```suggestion
    "scheduledWhen": "מועד תזמון",
```

Reply:
Will be resolved in https://github.com/apache/airflow/pull/51888